app/components/wds-global-navigation/search.js

```diff
@@ -1,24 +1,54 @@
 import {notEmpty, empty} from '@ember/object/computed';
 import Component from '@ember/component';
-import {computed} from '@ember/object';
+import EmberObject, {computed} from '@ember/object';
 import translations from 'npm:design-system-i18n/i18n/en/design-system';
```

Comment:
if you merged latest feature branch then this is not needed, just use i18n service to translate